build.sc

```diff
@@ -539,6 +581,27 @@ trait Cli extends SbtModule with ProtoBuildModule with CliLaunchers
   }
 }
 
+trait Cli3 extends Cli {
+  override def myScalaVersion = Scala.scala3
+
+  val argsFile = os.pwd / os.RelPath("out") / ".graal-cp-args"
+
+  override def nativeImageClassPath = T {
+    // TODO - results are cached - need to add inputs or caching method
+    val classpath     = super.nativeImageClassPath().map(_.path).mkString(File.pathSeparator)
+    val coursierCache = coursier.cache.FileCache[coursier.util.Task]().location.toString
```

Comment:
You shouldn't use the coursier cache path here… It's not meant to be used this way. `coursier.cache.shaded.dirs.ProjectDirectories` could be used instead (see `Directories.scala` for an example of use).